java/code/src/com/redhat/rhn/taskomatic/task/PackageCleanup.java

```diff
@@ -74,13 +77,29 @@ else if (log.isDebugEnabled()) {
 
             // Reset the queue (table)
             resetQueue();
+
+            // Change org for orphan vendor packages that a user can delete them
+            changeOrgForOrphanVendorPackages();
         }
         catch (Exception e) {
             log.error(e.getMessage(), e);
             throw new JobExecutionException(e);
         }
     }
 
+    private void changeOrgForOrphanVendorPackages() {
+        LocalDateTime now = LocalDateTime.now();
+        if (now.getHour() == 21 && now.getMinute() >= 49) {
+            // should be executed only at 21:50 every day
+            WriteMode update = ModeFactory.getWriteMode(TaskConstants.MODE_NAME,
+                    TaskConstants.TASK_QUERY_PKGCLEANUP_ORPHAN_VENDOR_PKG_CHANGE_ORG);
+            int updates = update.executeUpdate(Map.of("org_id", OrgFactory.getSatelliteOrg().getId()));
+            if (updates > 0) {
+                log.info("{} orphan vendor packages ready to being removed", updates);
```

Comment:
This seems wrong to me; orphaned vendor packages are ready for removal but this code doesn't remove them. What about something like this? (A better way might be to automatically remove the packages if that's what we wanted to do)

 ```suggestion
                 log.info("found {} orphaned vendor packages; remove them manually" updates);
```

Reply:
Well, "ready for being removed" is not grammatical. It's either "ready for removal" (probably what you want) or "being removed" (which to my eye is what's causing the confusion since it means the script is doing the removal).

Reply:
I have changed the text and don't talk about remove at all.